src/metal.cs

```diff
@@ -304,6 +350,21 @@ public partial interface MTLComputeCommandEncoder : MTLCommandEncoder {
 		[Abstract]
 		[Export ("setBytes:length:atIndex:")]
 		void SetBytes (IntPtr bytes, nuint length, nuint index);
+
+		[iOS (10,0), TV (10,0), NoWatch, Mac (10,12)]
+		// [Abstract] - should be abstract but would break the API.
```

Comment:
Minor but you might want to surround this inside a `XAMCORE_4_0` block not really urgent to do since we will need to audit any breaking changes in the future if `XAMCORE_4_0`  becomes a thing but hey you might save your/our future self some seconds/minutes :P